js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -603,7 +604,8 @@ const DropDownEditor = TextBox.inherit({
             showCloseButton: false,
             toolbarItems: this._getPopupToolbarItems(),
             onPositioned: this._popupPositionedHandler.bind(this),
-            fullScreen: false
+            fullScreen: false,
+            contentTemplate: null
```

Comment:
Was it made to avoid problems with the named template (`content`)? Should we add a test for ddEditor with nested dxTemplate (e.g. for KO)?

Reply:
@DokaRus, what should we check using this test? That named templates are not working in ddEditors except ddBox? Since this option is not documented for ddEditors, maybe the test will be redundant.

Reply:
>what should we check using this test?

We should check that named template (`content` in this case) does not replace `contentTemplate` of the nested Popup widget.
@San4es has resolved this confusing behavior at [R608](https://github.com/DevExpress/DevExtreme/pull/11927/files#diff-daa74ef81b21e0ce80dd1b08baaeeb9cR608) but without test this line can be reverted after some kind of refactoring or renovating.

Reply:
>Should we add a test for ddEditor with nested dxTemplate

Done